packages/ui/src/components/vuestic-mixins/ClickOutsideMixin/tests/ClickOutsideMixin.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { CreateElement } from 'vue'
+import { mount as vtuMount } from '@vue/test-utils'
+import ClickOutsideMixin, { ClickOutsideOptions } from '../ClickOutsideMixin'
+
+describe('ClickOutsideMixin', () => {
+  const Component = (h: CreateElement) =>
+    h('div', {
+      ref: 'clickOutsideRef',
+    })
+
+  const onClickOutside = jest.fn()
+
+  const options: ClickOutsideOptions = {
+    onClickOutside,
+    disabled: false,
+    trigger: 'click',
+  }
+
+  const baseComponentOptions: any = {
+    render: Component,
+    mixins: [ClickOutsideMixin],
+    mounted () {
+      const ref = this.$refs.clickOutsideRef
+
+      this.registerClickOutsideEvents(ref, this.$attrs)
+    },
+  }
+
+  const mount = (data: any) => {
```

Comment:
Small thing, but maybe keep original mount as it is, and rename your custom function?
This might confuse another developer.